Sources/PackageDescription/Version.swift

```diff
@@ -56,7 +56,11 @@ extension Version: Hashable {
         if let build = buildMetadataIdentifier {
             result = (result &* mul) ^ UInt64(build.hashValue)
         }
-        return Int(bitPattern: UInt(result))
+        #if arch(i386) || arch(arm)
```

Comment:
Yes, we definitely need hashing combiner support (in the standard library, really, but that is a huge proposal)... not on my short list of infrastructure to build though.